pluvo/pluvo.py

```diff
@@ -140,52 +139,44 @@ def _set_auth_params(self, params=None):
 
         return params
 
-    def _get(self, endpoint, params=None):
+    def _request(self, method, endpoint, data=None, params=None):
         headers = self._set_auth_headers()
         params = self._set_auth_params(params)
-
-        url = '{}{}'.format(self.api_url, endpoint)
-        r = requests.get(url, params=params, headers=headers)
-        data = r.json()
-
-        if r.status_code != 200:
-            raise PluvoAPIException(data['error'], r.status_code)
-
+        url = self.api_url + endpoint
+
+        r = requests.request(
+            method, url, params=params, json=data, headers=headers)
+        try:
+            data = r.json()
+        except ValueError:
+            if r.status_code == 500:
+                msg = ('Server returned a 500 error. This is likely a bug. '
+                       'Contact us at https://github.com/wendbv/pluvo/')
```

Comment:
This is not a public repo. Probably better to use https://github.com/wendbv/pluvo-python